website/content/docs/integration-guide.md

```diff
@@ -0,0 +1,46 @@
++++
+toc = true
+weight = 3
+draft = false
+title = "The integration guide"
+date = "2018-02-09T10:15:00+01:00"
+description = "Do you want to learn how to extend or integrate with bloop?"
+bref = "Learn how to integrate your own tooling with bloop."
++++
+
+## An overview of integrations
+
+There are two main ways you can extend bloop or integrate it with external
+tools. In this document, we dive into them and explain when you would want to
+use the first or the latter one.
+
+### Integrate via bloop's model
+
+Bloop has a well-defined, low-level configuration format that describes Scala
+and Java projects. This configuration format is the one that all bloop
+integrations in other build tools target.
+
+When generated, bloop picks up new files under bloop's project directory and
```

Comment:
When what is generated?

Reply:
Why is it important when it's generated?

Reply:
I think he meant "what" is generated?